iocore/net/UnixNet.cc

```diff
@@ -465,6 +433,9 @@ NetHandler::mainNetEvent(int event, Event *e)
     epd = (EventIO *)get_ev_data(pd, x);
     if (epd->type == EVENTIO_READWRITE_VC) {
       vc = epd->data.vc;
+      if (cop_list.in(vc)) {
+        cop_list.remove(vc);
+      }
```

Comment:
No bug and no issue here, the PR is only used to optimize InactivityCop.
Currently, I have no exactly numbers about this change, but the max concurrent connections test has been in the plan.